lib/menu.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'order'
+
+class Menu
```

Comment:
I was thinking that it might be possible to create a new instance of the menu class when you create a new order (in other words, we could inject Menu.new in to the order class). Just like you've done for @text = Text.new in the order class. 

This should mean that when we create a new order, it will have direct access to the menu. This will allow us to display the menu as a hash in the menu class, where the menu hash only contains dishes as the key and prices as the values (no mention of quantities). And then when we come to placing the order, we can view the menu items and prices, and then just add our chosen dishes and then select the quantity, which will be added to the order hash.  This should make the program run a bit more like it would in real life.